handlers/misttriggers/recording_end.go

```diff
@@ -153,16 +147,6 @@ func (d *MistCallbackHandlersCollection) triggerRecordingEndSegmenting(w http.Re
 			}
 		}()
 
-		// prepare .dtsh headers for all rendition playlists
-		for _, output := range outputs {
-			// output is multivariant playlist
-			err := createDtsh(requestID, output.Manifest)
-			if err != nil {
-				// should not block the ingestion flow or make it fail on error.
-				log.LogError(requestID, "master createDtsh() failed", err, "destination", output.Manifest)
-			}
-		}
-
```

Comment:
We still need this for mist playback. Am i missing some info?

Reply:
Note: MistInHLS needs to be fixed - dtsh for playlist is still broken which is why i removed it originally. Following up here: https://github.com/livepeer/catalyst-api/issues/178